src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/config.jelly

```diff
@@ -79,14 +79,13 @@
       <f:textbox />
     </f:entry>
 
-    <f:description>Number of executors per instance</f:description>
+    <f:description>Testing Number of executors per instance</f:description>
```

Comment:
Since numExecutors and executorScaler both determine the num of executors to configure, can we group them somehow just in the UI for clarity (without changing the implementation)?
FYI, I don't want you to change anything in the implementation as I would like to prioritize getting this out first and we can iterate on that later as needed.

Reply:
Discussed offline, letting it be as is for this feature.